reactfront/src/components/panels/sequence-panel.js

```diff
@@ -5,7 +5,11 @@ import {SafetyCard} from './safety-panel'
 
 export default function Sequences({ state, emit }) {
 
-    var sequences = (state.data && state.data.current_sequence) ?? []
+    var sequences = (state.data && state.data.current_sequence)
+    if (sequences === null || sequences === undefined) {
+        sequences = []
+    }
+
```

Comment:
This can be done more succinctly because JS will coerce "falsy" values like null and undefined to the boolean `false` when used as a condition. So you could say `var sequences = (state.data && state.data.current_sequence) || []`. It's a bit tricky to read granted, but `(a && b) || c` evaluates to `b` if `a` is true and `b` is not falsy (null/undefined/0/""/false/[]/{}/NaN), otherwise it evaluates to `c` as a fallback.

`a ?? b` is `a` if it's not null/undefined and `b` otherwise. `a || b` is `a` if it's not falsy and `b` otherwise. In this case we can use either because we only care about the null/undefined/[] cases, and `[] || []` is `[]`. Note we're using `a || b` to mean "`a` falling back on `b`" rather than the literal meaning "`a` boolean or `b`", a common trick in JS. Ditto with `a && b` - we use this to mean "`b` if `a` is true` and the falsy `a` otherwise